wordpress-mobile/WordPress-iOS
# Add signup analytics events

Fixes #8883

This PR adds a bunch of new analytics events, and brings us into sync with Android for this flow. This will give us deeper insight into how users signup within the app and make comparisons between platforms much more accurate.

To test:
Enter the signup flow and play around. You'll see events tracked in the Xcode console:

<img width="463" alt="screen shot 2018-04-19 at 12 05 46 pm" src="https://user-images.githubusercontent.com/517257/39010014-8c54e88e-43ca-11e8-8680-874d20c0bccd.png">

You can see the full list of events added in #8883 however it may be too exhausting to intentionally encounter every possibility covered. A thorough consideration of the code changes should be sufficient.

CC: @theck13 care to double check the events listed?
